.eslintrc

```diff
@@ -7,4 +7,7 @@
             "named": "never",
         }],
     },
+    "parserOptions": {
+        "ecmaVersion": 2017
```

Comment:
let’s enable this only for the tests that need it; otherwise breaking syntax could be added.

Reply:
I'll see if I can opt in on a per file basis, maybe a nested `.eslintrc` file in one of the test folders is sufficient.